geode-management/src/main/java/org/apache/geode/cache/configuration/GatewayReceiverConfig.java

```diff
@@ -61,7 +60,6 @@
  */
 @XmlAccessorType(XmlAccessType.FIELD)
 @XmlType(name = "", propOrder = {"gatewayTransportFilters"})
-@JsonIgnoreProperties(value = {"uri"}, allowGetters = true)
```

Comment:
Were these changes related to async operation framework or a different bug you found?

Reply:
kindof-unrelated cleanup, we found a way to annotate this in one place instead of having to add it to every new class, including some new classes in this PR.  went ahead and cleaned up the existing classes too.